cmd/storage-datatypes.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // DeleteOptions represents the disk level delete options available for the APIs
 type DeleteOptions struct {
-	Recursive bool `msg:"r"`
-	Force     bool `msg:"f"`
+	Recursive      bool `msg:"r"`
+	ImmediatePurge bool `msg:"ip"`
```

Comment:
If it isn't too much of a hazzle, I think "Immediate" is enough. "Purge" is implied by it being a delete function.
```suggestion
	Immediate bool `msg:"i"`
```